java/pom.xml

```diff
@@ -246,7 +246,8 @@
         <configuration>
           <java>
             <googleJavaFormat>
-              <version>1.7</version>
+              <!-- 1.19.1 support JDK21 -->
+              <version>1.19.1</version>
               <style>GOOGLE</style>
```

Comment:
I'll suggest we use the plantir style for longer line length and better breaking lines strategy.

Reply:
You can check this build https://github.com/apache/curator/blob/111f3a6bdcd562db0493e52243e5cec933cc36ef/pom.xml#L933-L954.

Reply:
Thanks, I will do this in another PR

Reply:
plantir style seems doesn't support 2 spaces: https://github.com/palantir/palantir-java-format/issues/951. I suggest continuing to use google code style